build/pipelines/azure-pipelines.release.yaml

```diff
@@ -8,7 +8,7 @@ trigger: none
 pr: none
 
 variables:
-  versionMajor: 11
+  versionMajor: 0
```

Comment:
Restore back to 11 when this PR is ready to check in?

Reply:
Nice reminding! The PR is ready for review now.